Sources/SmokeOperationsHTTP1/SmokeInvocationTraceContext.swift

```diff
@@ -90,10 +90,7 @@ extension SmokeInvocationTraceContext: OperationTraceContext {
         
         if let bodyData = bodyData {
             logMetadata["bodyBytesCount"] = "\(bodyData.count)"
-            
-            if logger.logLevel <= .debug {
```

Comment:
Wouldn't changing this to `.info` be enough? Because it seems removing this condition would log the requests/ response at higher levels

Reply:
Nevermind, just noticed the line at 97